uhdm-plugin/UhdmAst.cc

```diff
@@ -1086,15 +1086,26 @@ AST::AstNode *UhdmAst::process_value(vpiHandle obj_h)
         // so we are treating here UInt in the same way as if they would be Int
         case vpiUIntVal:
         case vpiIntVal: {
-            auto size = vpi_get(vpiSize, obj_h);
+            int size = -1;
+            bool is_signed = false;
+            visit_one_to_one({vpiTypespec}, obj_h, [&](AST::AstNode *node) {
+                if (node && node->attributes.count(UhdmAst::packed_ranges()) && node->attributes[UhdmAst::packed_ranges()]->children.size() &&
+                    node->attributes[UhdmAst::packed_ranges()]->children[0]->children.size()) {
+                    size = node->attributes[UhdmAst::packed_ranges()]->children[0]->children[0]->integer;
+                }
```

Comment:
Could you add a comment describing what is done here?
It's hard to guess what these objects are, for example `node->attributes[UhdmAst::packed_ranges()]->children[0]->children[0]`

Reply:
Added.